### PR TITLE
WIP:Format the go import statements

### PR DESCRIPTION
**Pull request by daniel-hutao**
Signed-off-by: daniel-hutao <farmer.hutao@outlook.com>

**What type of PR is this?**
 /kind flake

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for reviewers**:
```
```

**Additional documentation, usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by LinuxSuRen** [comment]
Thanks for the improvement. I got a different result, so I'm wondering if you can provide the command that you used for the format. Below is the output from my side:

```
➜  kubesphere git:(fix-pipe-input-admin) gh pr checkout 3606
remote: Enumerating objects: 89, done.
remote: Counting objects: 100% (89/89), done.
remote: Compressing objects: 100% (35/35), done.
remote: Total 89 (delta 43), reused 78 (delta 39), pack-reused 0
Unpacking objects: 100% (89/89), 20.88 KiB | 5.00 KiB/s, done.
From github.com:kubesphere/kubesphere
 * [new ref]           refs/pull/3606/head -> daniel-hutao/master
Switched to branch 'daniel-hutao/master'
➜  kubesphere git:(daniel-hutao/master) go fmt ./...
go: downloading github.com/go-resty/resty/v2 v2.5.0
kube/pkg/apis/core/helper/helpers.go
kube/pkg/quota/v1/evaluator/core/persistent_volume_claims.go
kube/plugin/pkg/admission/resourcequota/resource_access.go
➜  kubesphere git:(daniel-hutao/master) ✗ git status
On branch daniel-hutao/master
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   kube/pkg/apis/core/helper/helpers.go
	modified:   kube/pkg/quota/v1/evaluator/core/persistent_volume_claims.go
	modified:   kube/plugin/pkg/admission/resourcequota/resource_access.go

no changes added to commit (use "git add" and/or "git commit -a")
➜  kubesphere git:(daniel-hutao/master) ✗ git diff
➜  kubesphere git:(daniel-hutao/master) ✗ go version
go version go1.15.2 darwin/amd64
```

/kind cleanup

**Comment by yuswift** [comment]
I think there should only exist three parts in the `go import`. Firstly is the standard library, secondly is the third-part library, lastly is the library written by ourselves. But most of the files you changed have four parts. Could you merge them into three parts?

**Comment by LinuxSuRen** [comment]
I'd prefer to having a unified and easy way to format the code lines. The command `go fmt` comes from the Golang official. [A lot of open-sources projects](https://github.com/search?l=Makefile&q=%22go+fmt%22&type=Code) are using it.

See also https://stackoverflow.com/questions/47735678/goland-how-to-use-gofmt/47737130

**Comment by daniel-hutao** [comment]
> I think there should only exist three parts in the `go import`. Firstly is the standard library, secondly is the third-part library, lastly is the library written by ourselves. But most of the files you changed have four parts. Could you merge them into three parts?

??? four parts?

**Comment by daniel-hutao** [comment]
> I'd prefer to having a unified and easy way to format the code lines. The command `go fmt` comes from the Golang official. [A lot of open-sources projects](https://github.com/search?l=Makefile&q=%22go+fmt%22&type=Code) are using it.
> 
> See also https://stackoverflow.com/questions/47735678/goland-how-to-use-gofmt/47737130

I've took a look at https://stackoverflow.com/questions/47735678/goland-how-to-use-gofmt/47737130. And I almost did the same yesterday, and my goland is also configured “File Watchers”, in there it runs **go fmt** or **goimports** everytime I save a Go file. 

But go fmt and goimports two commands cannot divide import statement into three categories and sort them separately, so I manually divided them into three categories first, and then used the tools of goland to automatically complete the sorting and code formatting.

And I also think that this work should be done in a fully automatic and simple way. I will continue to try to find out if there is a corresponding method.

**Comment by LinuxSuRen** [comment]
Awesome! If you're going to continue work on this PR. I'd suggest that mark this PR as WIP(working in progress) until you believe it's ready for the review.

File: cmd/ks-apiserver/app/options/options.go [hunk 1]
```diff
@@ -42,8 +45,6 @@ import (
 	"kubesphere.io/kubesphere/pkg/simple/client/s3"
 	fakes3 "kubesphere.io/kubesphere/pkg/simple/client/s3/fake"
 	"kubesphere.io/kubesphere/pkg/simple/client/sonarqube"
```
Comment (yuswift) [inline cmd/ks-apiserver/app/options/options.go:47]: Some of the files have four parts. I used to merge them into three parts manually.

Comment (zryfish) [reply cmd/ks-apiserver/app/options/options.go:47]: I think we need a format guideline which all developers can follow, also it's useful when setting up CI.

**Comment by daniel-hutao** [comment]
> Thanks for the improvement. I got a different result, so I'm wondering if you can provide the command that you used for the format. Below is the output from my side:
> 
> ```
> ➜  kubesphere git:(fix-pipe-input-admin) gh pr checkout 3606
> remote: Enumerating objects: 89, done.
> remote: Counting objects: 100% (89/89), done.
> remote: Compressing objects: 100% (35/35), done.
> remote: Total 89 (delta 43), reused 78 (delta 39), pack-reused 0
> Unpacking objects: 100% (89/89), 20.88 KiB | 5.00 KiB/s, done.
> From github.com:kubesphere/kubesphere
>  * [new ref]           refs/pull/3606/head -> daniel-hutao/master
> Switched to branch 'daniel-hutao/master'
> ➜  kubesphere git:(daniel-hutao/master) go fmt ./...
> go: downloading github.com/go-resty/resty/v2 v2.5.0
> kube/pkg/apis/core/helper/helpers.go
> kube/pkg/quota/v1/evaluator/core/persistent_volume_claims.go
> kube/plugin/pkg/admission/resourcequota/resource_access.go
> ➜  kubesphere git:(daniel-hutao/master) ✗ git status
> On branch daniel-hutao/master
> Changes not staged for commit:
>   (use "git add <file>..." to update what will be committed)
>   (use "git restore <file>..." to discard changes in working directory)
> 	modified:   kube/pkg/apis/core/helper/helpers.go
> 	modified:   kube/pkg/quota/v1/evaluator/core/persistent_volume_claims.go
> 	modified:   kube/plugin/pkg/admission/resourcequota/resource_access.go
> 
> no changes added to commit (use "git add" and/or "git commit -a")
> ➜  kubesphere git:(daniel-hutao/master) ✗ git diff
> ➜  kubesphere git:(daniel-hutao/master) ✗ go version
> go version go1.15.2 darwin/amd64
> ```
> 
> /kind cleanup

@LinuxSuRen I've fixed the problem mentioned here. These problems are because some files have not been formatted before, and this pr is not involved it; but I have executed the global formatting commands *go fmt ./...* . I have not found a better way to automatically divide the import into 3 parts yet. I think we can merge this pr first. The automatic classification of import statements can be studied later.

**Comment by daniel-hutao** [comment]
I have found a better way to automatically divide the import into 3 parts:
`goimports -local=kubesphere.io/kubesphere -d -w`

**Comment by daniel-hutao** [comment]
I formatted the import statements with `goimports -local=kubesphere.io/kubesphere -d -w .` and found that there were more files adjusted than I thought. I have filtered out the modification of the vendor directory. @LinuxSuRen

**Comment by daniel-hutao** [comment]
kubesphere/kubesphere/pull/3625

**Comment by daniel-hutao** [comment]
Due to conflicts and corev1 related issues, it has been a bit confusing, so I reopened a pr /pull/3625

**Comment by daniel-hutao** [comment]
Due to conflicts and corev1 related issues, it has been a bit confusing, so I reopened a pr /pull/3625

**Pull request closed without merging**